integration_testing/features/coach/coach-create-groups.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Coach create at least 2 groups
+    Coach should be able to sign in to access Kolibri Coach tab
```

Comment:
As above, delete here and add as a `Given that I am signed in to Kolibri as a coach user` in the `Background`.